convert2rhel/checks.py

```diff
@@ -272,26 +278,32 @@ def check_custom_repos_are_valid():
 
 
 def ensure_compatibility_of_kmods():
-    """Ensure if the host kernel modules are compatible with RHEL."""
+    """Ensure that the host kernel modules are compatible with RHEL.
+
+    :raises SystemExit: Interrupts the conversion because some kernel modules are not supported in RHEL.
+    """
     host_kmods = get_loaded_kmods()
     rhel_supported_kmods = get_rhel_supported_kmods()
     unsupported_kmods = get_unsupported_kmods(host_kmods, rhel_supported_kmods)
-    if unsupported_kmods:
+
+    # Validate the best case first. If we don't have any unsupported_kmods, this means
+    # that everything is compatible and good to go.
+    if not unsupported_kmods:
+        logger.debug("All loaded kernel modules are available in RHEL.")
+    else:
         not_supported_kmods = "\n".join(
```

Comment:
The map/lambda code block is something I removed under the commit https://github.com/oamg/convert2rhel/commit/5f52da81b3e23fb6a03cb4e5c2f4d251cb6b700f. The commit has been dropped from the PR twice. Please apply the patch or let me know why is it better to keep the map/lambda then the list comprehension.

Reply:
You're correct, actually. The list comprehension is more readable. 

We were having some problems during the rebase/pull of this branch. I will make sure to update it on Monday.

Reply:
done.